auto_round_extension/qbits/qlinear_qbits.py

```diff
@@ -91,6 +95,20 @@ def __init__(
 
         self.trainable = trainable
 
+    def req_check(self):
+        torch_version = str(torch.__version__)
+        if QBITS_AVAILABLE:
+            itrex_version = str(intel_extension_for_transformers.__version__)
+            version_match_map = {"1.4": "2.2.0+cpu",
+                                "1.4.1": "2.2.0+cpu", "1.4.2": "2.3.0+cpu"}
+            if itrex_version in version_match_map:
+                if torch_version != version_match_map[itrex_version]:
+                    logger.warning(
+                        f"Intel Extension for Transformers {itrex_version} is not compatible with torch {torch_version}. Please install torch {version_match_map[itrex_version]}")
```

Comment:
could we just present the exact install cmd to  users,  format it as "Please install torch ....... as itrex ..."

Reply:
fix in b90606f